modules/custom-status/custom-status.php

```diff
@@ -459,747 +466,806 @@ function post_admin_header() {
-		if ( isset( $_GET['action'], $_GET['term-id'] ) && $_GET['action'] == 'edit-status' ): ?>
-		<?php
-			// Check whether the term exists
-			$term_id = (int)$_GET['term-id'];
-			$status = $this->get_custom_status_by( 'id', $term_id  );
-			if ( !$status ) {
-				echo '<div class="error"><p>' . $this->module->messages['status-missing'] . '</p></div>';
-				return;
-			}
-			$edit_status_link = $this->get_link( array( 'action' => 'edit-status', 'term-id' => $term_id ) );
+		/**
+		 * Primary configuration page for custom status class.
+		 * Shows form to add new custom statuses on the left and a
+		 * WP_List_Table with the custom status terms on the right
+		 *
+		 * Disabling nonce verification because that is not available here, it's just rendering it. The actual save is done in helper_settings_validate_and_save and that's guarded well.
+		 * phpcs:disable:WordPress.Security.NonceVerification.Missing
+		 */
+		public function print_configure_view() {
+			global $edit_flow;
+
+			/** Full width view for editing a custom status **/
+			if ( isset( $_GET['action'], $_GET['term-id'] ) && 'edit-status' == $_GET['action'] ) :
+				?>
+				<?php
+				// Check whether the term exists
+				$term_id = (int) $_GET['term-id'];
+				$status = $this->get_custom_status_by( 'id', $term_id );
+				if ( ! $status ) {
+					echo '<div class="error"><p>' . esc_html( $this->module->messages['status-missing'] ) . '</p></div>';
+					return;
+				}
+				$edit_status_link = $this->get_link( array(
+					'action' => 'edit-status',
+					'term-id' => $term_id,
+				) );
 
-			$name = ( isset( $_POST['name'] ) ) ? stripslashes( $_POST['name'] ) : $status->name;
-			$description = ( isset( $_POST['description'] ) ) ? strip_tags( stripslashes( $_POST['description'] ) ) : $status->description;
-		?>
+				$name = ( isset( $_POST['name'] ) ) ? stripslashes( $_POST['name'] ) : $status->name;
+				$description = ( isset( $_POST['description'] ) ) ? strip_tags( stripslashes( $_POST['description'] ) ) : $status->description;
+				?>
 
 		<div id="ajax-response"></div>
-		<form method="post" action="<?php echo esc_attr( $edit_status_link ); ?>" >
+		<form method="post" action="<?php echo esc_url( $edit_status_link ); ?>" >
 		<input type="hidden" name="term-id" value="<?php echo esc_attr( $term_id ); ?>" />
-		<?php
-			wp_original_referer_field();
-			wp_nonce_field( 'edit-status' );
-		?>
+				<?php
+				wp_original_referer_field();
+				wp_nonce_field( 'edit-status' );
+				?>
 		<table class="form-table">
 			<tr class="form-field form-required">
 				<th scope="row" valign="top"><label for="name"><?php _e( 'Custom Status', 'edit-flow' ); ?></label></th>
-				<td><input name="name" id="name" type="text" value="<?php echo esc_attr( $name ); ?>" size="40" aria-required="true" <?php if( 'draft' === $status->slug ) echo 'readonly="readonly"' ?> />
+				<td><input name="name" id="name" type="text" value="<?php echo esc_attr( $name ); ?>" size="40" aria-required="true"
```

Comment:
Same suggestion as the other multi-line class, I'd recommend [pre-computing the extra attribute](https://github.com/Automattic/Edit-Flow/pull/734#discussion_r1655490459) and echoing, or using a ternary instead.

Reply:
Done